functions/index.js

```diff
@@ -1,7 +1,53 @@
-const functions = require('firebase-functions');
-var cors = require('cors')({ origin: true });
 
-const admin = require('firebase-admin');
-admin.initializeApp();
```

Comment:
Please check according to the update important lines are removed.

Reply:
okay